744_find_smallest_letter.py

```diff
@@ -0,0 +1,13 @@
+from typing import List
+
+
+class Solution:
```

Comment:
You can solve this problem by a more efficient solution.

Reply:
I tried the different way. Please check.